openquake/hazardlib/geo/surface/base.py

```diff
@@ -431,21 +431,29 @@ def get_middle_point(self):
 
         return mesh.get_middle_point()
 
-    def get_resampled_top_edge(self, angle_var=0.1):
+    def get_resampled_top_edge(self, return_top_edge_index=False, angle_var=0.1):
         """
```

Comment:
I can't recall why I made this change before making one more possible output of index of top edge nodes in the mesh.
This function hasn't been used. 
I will leave it for now. I will delete it later if I still forget the reason on making this change.